### ISSUE TITLE
ImageGrinder plugin (com.diffplug.gradle.image-grinder) version 2.1.2 fails to run on Gradle 6.2

### ISSUE THREAD

**Issue by quittle** [diffplug/image-grinder#5]
I recently tried upgrading my project to Gradle 6.2 (both `6.2` and `6.2.2`) and ran across this error with the latest version of your plugin (`2.1.2`). Perhaps this method was deprecated in Gradle 6.2

```
java.lang.NoSuchMethodError: org.gradle.util.SingleMessageLogger.nagUserOfReplacedPlugin(Ljava/lang/String;Ljava/lang/String;)V
        at com.diffplug.gradle.imagegrinder.LegacyPlugin.apply(LegacyPlugin.java:43)
        at com.diffplug.gradle.imagegrinder.LegacyPlugin.apply(LegacyPlugin.java:25)
        at org.gradle.api.internal.plugins.ImperativeOnlyPluginTarget.applyImperative(ImperativeOnlyPluginTarget.java:43)
...
```

**Comment by nedtwigg** [comment]
Thanks for reporting!  A quick workaround is to use `com.diffplug.image-grinder`, rather than `com.diffplug.gradle.image-grinder`.

**Comment by quittle** [comment]
I have worked around it with the plugin change you suggested. Is there any benefit of using `com.diffplug.gradle.image-grinder` over `com.diffplug.image-grinder`? It's not clear to me what the differences are.

**Comment by nedtwigg** [comment]
The `NoSuchMethodError` is hiding a nice deprecation error which describes that it's just a name change, no behavior change, [for this reason](https://dev.to/nedtwigg/names-in-java-maven-and-gradle-2fm2#gradle-plugin-id).

**Comment by quittle** [comment]
Ah, that is some unfortunate irony. Thank you for explaining

**Comment by nedtwigg** [comment]
Fixed by 1c3736e66729fdf260cf8652102382ca910f7c6f
